src/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ option(BUILD_MODULES "Build OSConfig Modules" ON)
 option(BUILD_PLATFORM "Build OSConfig Platform" ON)
 option(BUILD_TESTS "Build test collateral" ON)
 option(BUILD_SAMPLES "Build samples" OFF)
+option(BUILD_Rust "Build Rust Modules" OFF)

```

Comment:
Maybe `BUILD_RUST` instead of `BUILD_Rust` to be consistent with the other options?